keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/DoubleVertexSamples.java

```diff
@@ -39,4 +39,7 @@ public DoubleTensor getVariances() {
             .divInPlace(samples.size()-1);
     }
 
+    public DoubleTensor asTensor() {
+        return DoubleTensor.pile(samples.toArray(new DoubleTensor[0]));
```

Comment:
I think we agreed previously (I think at your instigation??) that `new DoubleTensor[0]` is esoteric and it's clearer to do `samples.stream().toArray(DoubleTensor[]::new)`